### ISSUE TITLE
"Disable Screen Dimming" does not work on iPhone

### ISSUE THREAD

**Issue by chchen-stanford** [ish-app/ish#1491]
In the process of installing texlive-full, as discussed in [#1056](https://github.com/ish-app/ish/issues/1056#issuecomment-878567756), I found that the "Disable Screen Dimming" setting does not work on iPhone 6s. It worked fine on iPad 7th. Both are on iOS 14.6.

The "Disable Screen Dimming" setting is critical for installing large packages. I ended up disabling the screen "Auto-Lock" in the iPhone system settings. Otherwise, I kept getting "I/O error" when the screen is locked. 

By the way, iSH is truly amazing. Thank you for such a great app!

**Comment by guanzhangrtk** [comment]
Does this still happen if you keep the app running in the background eg. `cat /dev/location > /dev/null &`?

**Comment by tbodt** [comment]
Previously: #784 #1242 

This could be an iOS bug, in which case I can't do anything about it. It might also be that "don't turn off the screen" is more of a guideline than a rule.

**Comment by tbodt** [comment]
Duplicate of #784

**Comment by chchen-stanford** [comment]
> Do you still get the I/O errors if you keep the app running in the background eg. `cat /dev/location > /dev/null &`?

I am not well versed in Linux to understand this suggestion. A related question though: can iSH be programmed to run uninterrupted in the background? Other apps like a-Shell seem to be able to run in the background fine.

**Comment by 62f** [comment]
@chchen-stanford this suggestion concerns not a linux solution but rather of an attempted iOS-based override of iOS’s battery/screen/DRAM-saving function.

**Comment by chchen-stanford** [comment]
I have no prior experience with Linux and use iSH mainly to process TeX files locally. iSH is great in offering a complete Linux system. However, iSH has room for improvement in terms of user experience as an iOS app. Here is a list of surprises to me as a Linux layman:
1. iSH sometimes stops working if not in the foreground. (I learned to keep the iSH screen active if I install any packages.) The "disable screen dimming" option is offered, but not always functioning.
2. Local network access is needed for any internet access. In contrast, the local network access is disabled for all other iOS apps I use, unless I need two devices to talk to each other on the local network.
3. Texlive-full package does not install properly. Although I eventually figured out a workaround of installing texlive and then texlive-full, the full package is still only partially functioning. Because texlive-full is very large, it is particularly challenging for trial-and-error installations. I do understand that iSH has very little control over the Alpine texlive distribution. 

@tbodt I am not sure if it is appropriate to post my user experience here, but it might help other users like me. Despite all these things, iSH is still an outstanding accomplishment, and I owe it to you to have a mobile yet local TeX processor.
